WORKSPACE

```diff
@@ -165,6 +165,10 @@ KOTLINX_COROUTINES_VERSION = "1.3.3"
 
 ROBOLECTRIC_VERSION = "4.1"
 
+KOTLINPOET_VERSION = "1.5.0"
```

Comment:
Internal version is 1.0.1

Reply:
Ah, thanks for checking. I'll update this.